Minigames/src/main/java/au/com/mineauz/minigames/managers/PlaceHolderManager.java

```diff
@@ -0,0 +1,185 @@
+package au.com.mineauz.minigames.managers;
+
+import au.com.mineauz.minigames.Minigames;
+import au.com.mineauz.minigames.minigame.Minigame;
+import au.com.mineauz.minigames.objects.MinigamePlayer;
+import au.com.mineauz.minigames.objects.ModulePlaceHolderProvider;
+import me.clip.placeholderapi.expansion.PlaceholderExpansion;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Player;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+
+
+/**
+ * Created for the AddstarMC Project.
+ * Created by Narimm on 3/06/2020.
+ */
+public class PlaceHolderManager extends  PlaceholderExpansion  {
+
+    private final Minigames plugin;
+    private final List<ModulePlaceHolderProvider> providers;
+    private final Map<String,String> identifiers;
+
+    public PlaceHolderManager(Minigames plugin) {
+        this.plugin = plugin;
+        providers = new ArrayList<>();
+        identifiers = new HashMap<>();
+        identifiers.put("gameCount","CORE");
+        identifiers.put("enabledGameCount","CORE");
+        identifiers.put("totalPlaying","CORE");
+    }
+
+    public Set<String> getRegisteredPlaceHolders() {
+        return identifiers.keySet();
+    }
+
+    @Override
+    public String onRequest(OfflinePlayer p, String params) {
+        return super.onRequest(p, params);
+    }
+
+    @Override
+    public boolean canRegister(){
+        return true;
+    }
+
+    @Override
+    public boolean persist(){
+        return true;
+    }
+    @Override
+    public String getIdentifier() {
+        return Minigames.getPlugin().getName();
+    }
+
+    @Override
+    public String getAuthor() {
+        return Minigames.getPlugin().getDescription().getAuthors().toString();
+    }
+
+    @Override
+    public String getVersion() {
+        return Minigames.getVERSION().toString();
+    }
+
+    @Override
+    public String onPlaceholderRequest(Player player, String identifier){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'onPlaceholderRequest(Player, String)' has an NPath complexity of 272](https://app.codacy.com/manual/Narimm/Minigames/pullRequest?prid=5629519)